sched/irq/irq_csection.c

```diff
@@ -232,7 +232,7 @@ irqstate_t enter_critical_section(void)
 
           DEBUGASSERT((g_cpu_irqset & (1 << cpu)) == 0);
 
-          spin_lock(&g_cpu_irqlock);
+          spin_lock_wo_note(&g_cpu_irqlock);
```

Comment:
do we need change leave_critical_section too

Reply:
leave_critical_section  use  cpu_irqlock_clear which alse use spin_unlock_wo_note
![image](https://github.com/user-attachments/assets/af7dbdd7-1265-4574-bd36-7f99b17c9e1c)

Reply:
ok